the-internet/pom.xml

```diff
@@ -31,9 +31,9 @@
 			<version>7.1.0</version>
```

Comment:
See bottom of the PR. Some files appear to be in binary or some encoding that the comparer cannot interpret. I'll add my comments for those files here at the top.